idea/testData/codeInsight/surroundWith/if/moveDeclarationsOut/order/valAndClass.kt.after

```diff
@@ -3,9 +3,9 @@ fun foo() {
         fun test() {}
     }
 
-    val d: A
+    val d =
     if (<caret>) {
-        d = A()
+        A()
```

Comment:
I don't think this new line is the problem with this pull request.

If I have following case

```kotlin
fun foo() {
    <selection>class A {
        fun test() {}
    }
    
    
    
    val d: A = A()</selection>

    d.test()
    A()
}
```

For current kotlin plugin converts them into 

```kotlin
class A {
        fun test() {
        }
    }

    val d: A


    if () {
        d = A()
    }
```